src/interface_adapter/SearchByIdPresenter.java

```diff
@@ -0,0 +1,37 @@
+package interface_adapter;
+
+import use_case.search_by_id.SearchIdOutputBoundary;
+import use_case.search_by_id.SearchIdOutputData;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
```

Comment:
The java.time.LocalDateTime and java.time.format.DateTimeFormatter imports can be removed as they don't seem to be used anywhere.